collector/src/execute.rs

```diff
@@ -110,6 +110,19 @@ fn default_runs() -> usize {
     3
 }
 
+#[derive(Debug, Clone, serde::Deserialize)]
+#[serde(rename_all = "lowercase")]
+enum BenchmarkCategory {
```

Comment:
Nit: just call it `Category`/`category` throughout, it's shorter and no less meaningful given that there aren't other kinds of category :)